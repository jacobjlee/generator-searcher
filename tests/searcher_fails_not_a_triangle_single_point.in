0.10, 1.00
