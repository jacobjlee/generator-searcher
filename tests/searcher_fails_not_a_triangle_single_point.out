read 1 points
0.10, 1.00
This is not a triangle
